docs/writing-tests/interaction-testing.mdx

```diff
@@ -56,24 +56,71 @@ Once the story loads in the UI, it simulates the user's behavior and verifies th
 
 <Video src="../_assets/writing-tests/addon-interaction-example-optimized.mp4" />
 
-### Run code before each test
+<If notRenderer="angular">
```

Comment:
I opted to conditionally render this content so that Angular viewers don't even see this feature they cannot use.

But should we render fallback content showing them how to use story-level `beforeEach`? @kasperpeulen

Reply:
I like this! Yes, `beforeEach` should be shown for Angular